### PR TITLE
Variable to integer and output modification

### PR DESCRIPTION
**Pull request by Biohazardhpk**
---

### Step template checklist

- [x] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [x] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [x] Parameter names should not start with `$`
- [x] **Step template parameter names (the ones declared in the JSON, not the script body) should be prefixed with a namespace so that they are less likely to clash with other user-defined variables in Octopus** (see [this issue](https://github.com/OctopusDeploy/Issues/issues/2126)). For example, use an abbreviated name of the step template or the category of the step template).
- [x] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [x] If a new `Category` has been created:
   - [x] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [x] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

Fixes # . _If there is an open issue that this PR fixes add it here, otherwise just remove this line_

### CODE REVIEW

**Comment by matt-richardson** [comment]
Hi @Biohazardhpk - thanks for the PR.

The `write-verbose` calls are showing when log level is set to `verbose` in the deployment log view. I'm not sure this should be changed, as it could massively increase the size of the log for some customers. I'd suggest adding a `write-output` before the loop, that says `"Waiting for $seconds seconds"` and potentially another one afterwards.

Also, please review the "Step template checklist" above - the `Version` of the template should be updated, otherwise it wont sync. You may also want to consider updating the `LastModifiedOn` date.

File: step-templates/wait.json [hunk 1]
```diff
@@ -3,11 +3,11 @@
   "Name": "Wait",
   "Description": "Pauses the process for a set duration",
   "ActionType": "Octopus.Script",
-  "Version": 1,
+  "Version": 1.1,
```
Comment (tothegills) [inline step-templates/wait.json]:
```suggestion
  "Version": 2,
```

File: step-templates/wait.json [hunk 2]
```diff
@@ -3,11 +3,11 @@
   "Name": "Wait",
   "Description": "Pauses the process for a set duration",
   "ActionType": "Octopus.Script",
-  "Version": 1,
+  "Version": 2,
   "CommunityActionTemplateId": null,
   "Packages": [],
   "Properties": {
-    "Octopus.Action.Script.ScriptBody": "$Wait = $Seconds\nfor ($CountDown = $Wait; $CountDown -ge 0; $CountDown--){\nWrite-Verbose \"$CountDown seconds remaining\"\nStart-Sleep 1\n}",
+    "Octopus.Action.Script.ScriptBody": "[int]$Wait = $Seconds\nWrite-Output "Waiting for $seconds seconds"\nfor ($CountDown = $Wait; $CountDown -ge 0; $CountDown--){\nWrite-Verbose \"$CountDown seconds remaining\"\nStart-Sleep 1\n}",
```
Comment (ryanrousseau) [inline step-templates/wait.json:10]:
```suggestion
    "Octopus.Action.Script.ScriptBody": "[int]$Wait = $Seconds\nWrite-Output \"Waiting for $seconds seconds\"\nfor ($CountDown = $Wait; $CountDown -ge 0; $CountDown--){\nWrite-Verbose \"$CountDown seconds remaining\"\nStart-Sleep 1\n}",
```

Comment (ryanrousseau) [reply step-templates/wait.json:10]: @Biohazardhpk We need to escape the "s here. If you're good with the change, we can commit and merge the PR.

Comment (DevOpsDerek) [reply step-templates/wait.json:10]: Hi @Biohazardhpk, 

I wanted to check in and see if this change is ok to be made and we can publish the step?

Thanks

**Comment by Biohazardhpk** [comment]
Sure!

Eduard@mobile

**Pull request merged**